src/parser/ast/mod.rs

```diff
@@ -215,19 +215,41 @@ pub struct Use {
 pub enum Expression {
     // eval("$a = 1")
     Eval {
-        value: Box<Self>,
+        eval: Span,       // eval
+        start: Span,      // `(`
+        value: Box<Self>, // "$a = 1"
+        end: Span,        // `)`
+    },
+    // empty($a)
+    Empty {
+        empty: Span,      // empty
+        start: Span,      // `(`
+        value: Box<Self>, // $a
+        end: Span,        // `)`
     },
     // die, die(1)
     Die {
-        value: Option<Box<Self>>,
+        die: Span,                // die
+        start: Option<Span>,      // `(`
+        value: Option<Box<Self>>, // 1
+        end: Option<Span>,        // `)`
     },
     // exit, exit(1)
     Exit {
-        value: Option<Box<Self>>,
+        exit: Span,               // exit
+        start: Option<Span>,      // `(`
+        value: Option<Box<Self>>, // 1
+        end: Option<Span>,        // `)`
```

Comment:
maybe it makes more sense to have `arguments: Option<ArgumentList>` here?

Reply:
If using `arguments: Option<ArgumentList>` won't be an issue because it will only ever have zero or one argument, I agree that this change will make the code more elegant. I will make this change. =)